assets/.env

```diff
@@ -0,0 +1 @@
+WANDA_URL=http://localhost:4001
```

Comment:
was this file uploaded in purpose? 
I would actually put the `.env` file in the gitignore. Having the example file is good enough, so everyone can configure their own environment